Sources/ProjectSpec/TargetSource.swift

```diff
@@ -4,8 +4,18 @@ import PathKit
 
 public struct TargetSource: Equatable {
     public static let optionalDefault = false
+    
+    private var _path: String = ""
 
-    public var path: String
+    public var path: String {
+        get {
+            return _path
+        }
+        set {
+            _path = TargetSource.clean(path: newValue)
```

Comment:
What about sanitizing using `didSet`? That would avoid using a backing `_path` variable.

Reply:
Honest answer? I tried and Xcode shouted at me for doing it wrong. It's been about a year since I last wrote any Swift code so I assumed I'd just mis-remembered :D  

I'll take another look.